book/src/wasm_limitations.md

```diff
@@ -0,0 +1,26 @@
+# Limitations of WASM
+
+- Safari cannot spawn nested `Worker`s. To get around this limitation, build two variants of the library,
+  one with multithreading and one without, and serve Safari users the single-threaded variant.
+  For a more general solution, check out [wasm-feature-detect](https://github.com/GoogleChromeLabs/wasm-feature-detect).
+- `std::thread::spawn` and replacements (e.g. `wasm_thread`) are not fully supported. This library includes
```

Comment:
So, does it mean that we cannot use std spawn, but need to use the macro at all places?

Reply:
`std::thread::spawn` *throws* on the web, so unless the user does not intend to target WASM they should always use `spawn!`.